src/search/request.rs

```diff
@@ -71,6 +71,10 @@ pub struct Search {
 
     #[serde(skip_serializing_if = "ShouldSkip::should_skip")]
     timeout: Option<Time>,
+
+    #[serde(flatten)]
+    #[serde(skip_serializing_if = "Map::is_empty")]
```

Comment:
```suggestion
    #[serde(skip_serializing_if = "ShouldSkip::should_skip")]
```

To be consistent with the rest of the crate

Reply:
Ah, I didn't notice that it's implemented for `Map`.

Reply:
Right there https://github.com/vinted/elasticsearch-dsl-rs/blob/706a5f5d0839fb586235e4b12c0b21500abecfed/src/util/should_skip.rs#L70-L80 👍